ui/app/templates/components/auth-form.hbs

```diff
@@ -26,6 +26,10 @@
       @errorMessage={{if (and cluster.standby hasCSPError) cspErrorText error}}
       data-test-auth-error
     />
+    <div class="has-bottom-margin-s ">
+      <p class="is-label"> {{this.selectedAuthBackend.path}}</p>
```

Comment:
super small comment, but no spaces after the closing element.  In the line below too, I noticed a space before and after the opening and closing span tags.  e.g `>space{{this.selectedAuthBackend....`

Reply:
@Monkeychip - thank you for pointing this out! I wasn't sure which was correct 😅